rdmd_test.d

```diff
@@ -55,7 +55,7 @@ int main(string[] args)
     string testCompilerList; // e.g. "ldmd2,gdmd" (comma-separated list of compiler names)
 
     auto helpInfo = getopt(args,
-        "rdmd-default-compiler", "[REQUIRED] default D compiler used by rdmd executable", &defaultCompiler,
+        "D|default-compiler", "[REQUIRED] default D compiler used by rdmd executable", &defaultCompiler,
```

Comment:
Seems to the only capital option ... maybe consider choosing a lower-case letter ?

Reply:
I'm happy to take suggestions.  Although it did seem rather appropriate that it be `D` ;-)